public/services.js

```diff
@@ -1,10 +1,15 @@
 angular.module('services', [])
 .factory('userFactory', function ($http) {
 
-  var addStarterPokemon = function (pokemon) {
+  var addStarterPokemon = function (gameId, userId, pokemon) {
     return $http({
       method: 'PUT',
-      url: 'api/game/addpokemon'
+      url: 'api/games/addpokemon',
+      data: {
```

Comment:
If gameId, userId, & pokemon is passed in a data object
Will it available in the req.body?  The server side is expecting the info in the body